weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/CompareEvaluationsPage.tsx

```diff
@@ -237,3 +241,49 @@ const ResultExplorer: React.FC<{state: EvaluationComparisonState}> = ({
     </VerticalBox>
   );
 };
+
+const InvalidEvaluationBanner: React.FC<{state: EvaluationComparisonState}> = ({
+  state,
+}) => {
+  const [dismissed, setDismissed] = useState(false);
+  const invalidEvals = useMemo(() => {
+    return Object.values(state.data.evaluationCalls)
+      .filter(evalCall => {
+        return Object.keys(evalCall.summaryMetrics).length === 0;
```

Comment:
For readability, maybe define this as a short `isValidEval` helper function with a comment about why summaryMetrics indicates that?